index.html

```diff
@@ -721,6 +721,62 @@ <h2>
             </p>
           </div>
         </section>
+        <section>
+          <h2><dfn>CurrentTabPreferenceEnum</dfn></h2>
+          <p>
+            Recall that when offering the user a choice of [=display surfaces=]
+            to capture, the user agent MAY offer the [=browser=] [=display surface=]
+            in which the application is running. An application might prefer that
+            this particular [=display surface=] not be offered to the user.
+            The CurrentTabPreferenceEnum enumeration describes the different
+            preferences an application might have.
```

Comment:
I'd shorten this to not repeat normative language below. How about:
```suggestion
            Describes the different hints an application can provide about whether the
            tab the application is in should be among the choices offered to the user.
```